opentelemetry-proto local own dependency not found

**Describe your environment**

* Ubuntu 20.04 cmake version `3.16.3`  
* RHEL 8.5 cmake version `3.20.2-4.el8`
* Ubuntu 20.04 cmake version `3.25.0` (latest as of today) 

Tested with opentelemetry-cpp [latest](https://github.com/open-telemetry/opentelemetry-cpp/commit/7e26a7e84e94a50a0b04e70ed88ac04d75afbf40) (as of today) and opentelemetry-proto [v0.19.0](https://github.com/open-telemetry/opentelemetry-proto/tree/v0.19.0)

**Steps to reproduce**
Building with `OTELCPP_PROTO_PATH` set

```
cmake -DOTELCPP_PROTO_PATH=/path/to/proto ..
```

**What is the expected behavior?**
local opentelemetry dependency is found and used to build opentelemetry-cpp

**What is the actual behavior?**
it always fails with "not existing" error

```
CMake Error at cmake/opentelemetry-proto.cmake:30 (message):
  OTELCPP_PROTO_PATH does not point to a opentelemetry-proto repository
Call Stack (most recent call first):
  CMakeLists.txt:328 (include)
```

**Additional context**
Add any other context about the problem here.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,8 @@ Increment the:
 
 ## [Unreleased]
 
+* [BUILD] Fix opentelemetry-proto file exists check [#1824](https://github.com/open-telemetry/opentelemetry-cpp/pull/1824)
+
 ## [1.8.0] 2022-11-27
 
 * [DOC] Update Metrics status in README.md [#1722](https://github.com/open-telemetry/opentelemetry-cpp/pull/1722)

--- a/cmake/opentelemetry-proto.cmake
+++ b/cmake/opentelemetry-proto.cmake
@@ -26,7 +26,7 @@
 #
 
 if(OTELCPP_PROTO_PATH)
-  if(NOT EXISTS(${OTELCPP_PROTO_PATH}/opentelemetry/proto/common/v1/common.proto))
+  if(NOT EXISTS "${OTELCPP_PROTO_PATH}/opentelemetry/proto/common/v1/common.proto")
     message(FATAL_ERROR "OTELCPP_PROTO_PATH does not point to a opentelemetry-proto repository")
   endif()
   message(STATUS "opentelemetry-proto dependency satisfied by: external path")

